spec/requests/api/v1/customers/request_spec.rb

```diff
@@ -145,7 +145,7 @@
   end
 
   it "finds all customers matching created_at timestamps" do
-    create_list(:customer, 3)
+    create_list(:customer, 3, created_at: Time.now)
```

Comment:
@sallymacnicholas : before we were just generating three customers, and most of the time the tests would pass. Every 1 in 5 or so, the times would be different, and only return one or two. Adding the fixed Time.now works, but is there a better way?